guides/Shopify.md

```diff
@@ -0,0 +1,53 @@
+# Setting deploys to Shopify via CircleCI
+
+## Problem:
+
+By the reason of specificity of work with Shopify via `grunt-shopify`, we uploading changed file each time we save it. This may cause situations, when we push some changes to the server, then discard them with turned off `grunt`. So, we have some changes on the server and no records in git about them.  
```

Comment:
The primary challenge we faced when developing for Shopify was that the codebase checked in to Git was getting out of sync with the codebase deployed to Shopify. This issue was compounded when using the same theme on multiple stores e.g. for simple staging + production environments or in more complex, multi-geo scenarios.